### PR TITLE
feat: [STO-5203]: add STO to ServiceVersions

### PR DESCRIPTION
**Pull request by atmollo-harness**
### Summary

Add STO services (sto core and sto manager) to list in account page on NG

image

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by atmollo-harness** [comment]
retrigger titlecheck

**Comment by atmollo-harness** [comment]
retrigger cypress-rest

**Comment by atmollo-harness** [comment]
retrigger cypress-cd

**Comment by atmollo-harness** [comment]
retrigger cypress-pipeline

**Comment by atmollo-harness** [comment]
retrigger cypress-cv

**Review by pankajsingh-harness — DISMISSED** [review]

**Comment by atmollo-harness** [comment]
retrigger cypress-pipeline

File: src/modules/33-auth-settings/pages/AccountOverview/__test__/ServiceVersions.test.tsx [hunk 1]
```diff
@@ -10,6 +17,8 @@ global.fetch = jest.fn().mockImplementation(url => {
     versionObj.version = '1.0.0'
   } else if (url.includes('ccm/api/version')) {
     versionObj.versionInfo = '2.0.0'
+  } else if (url.includes('sto/api/v2/system/version') || url.includes('sto-manager/version')) {
+    versionObj.versionInfo = '1.0.0'
```
Comment (chris-harness) [inline src/modules/33-auth-settings/pages/AccountOverview/__test__/ServiceVersions.test.tsx:21]: Use a different version number to differentiate. Use different versions for sto and sto-manager as well

Comment (atmollo-harness) [reply src/modules/33-auth-settings/pages/AccountOverview/__test__/ServiceVersions.test.tsx]: fixed

**Comment by atmollo-harness** [comment]
retrigger jest

File: src/modules/33-auth-settings/pages/AccountOverview/__test__/ServiceVersions.test.tsx [hunk 2]
```diff
@@ -17,8 +17,10 @@ global.fetch = jest.fn().mockImplementation(url => {
     versionObj.version = '1.0.0'
   } else if (url.includes('ccm/api/version')) {
     versionObj.versionInfo = '2.0.0'
-  } else if (url.includes('sto/api/v2/system/version') || url.includes('sto-manager/version')) {
-    versionObj.versionInfo = '1.0.0'
+  } else if (url.includes('sto/api/v2/system/version')) {
+    versionObj.versionInfo = '3.0.0'
```
Comment (chris-harness) [inline src/modules/33-auth-settings/pages/AccountOverview/__test__/ServiceVersions.test.tsx:21]: `3.0.0` is already used below, so use a different version

Comment (atmollo-harness) [reply src/modules/33-auth-settings/pages/AccountOverview/__test__/ServiceVersions.test.tsx]: changed

**Comment by atmollo-harness** [comment]
retrigger jest

**Review by chris-harness — APPROVED** [review]

**Review by r0610205 — APPROVED** [review]

**Pull request merged**